go-garbage-collector/pxl_scripts/uprobe_scripts/GC.pxl

```diff
@@ -0,0 +1,28 @@
+import px
+import pxtrace
+
+pod = '<FILL_IN>'
```

Comment:
```suggestion
# Pixie references pods in the <namespace>/<pod> format. 
# E.g. pod = 'default/gc-service-fb666c487-4cjb7'
pod = '<FILL_IN>'
```

Reply:
I would add a comment in all the PxL files above the pod variable with the following in case they don't read step 4 closely.